contracts/cw20-escrow/src/contract.rs

```diff
@@ -98,6 +98,11 @@ pub fn try_cw20_create<S: Storage, A: Api, Q: Querier>(
     token: Cw20Coin,
     msg: CreateMsg,
 ) -> StdResult<HandleResponse> {
+    if token.is_empty() {
```

Comment:
Good check on the receiver (and needed).

Can we also add a check to cw20-base that we don't even send a 0 value? We cannot rely on it here, but nice check there as well.

Reply:
Sure. I will create the issue.

Reply:
Issue #89.